contrib/mesos/docs/architecture.md

```diff
@@ -37,6 +37,8 @@ The scheduling of a pod on Kubernetes on Mesos is essentially a two-phase proces
     - The k8sm-executor sends the pod to the kubelet which is part of the k8sm-executor process.
     - The kubelet launches the containers using Docker.
 
+![Scheduler Structure](scheduler.png)
```

Comment:
this is a pretty low level diagram to include here, maybe just have a text blurb that links to scheduler.md and drop the png in that doc